prime-router/metadata/tables/livd/LIVD-Supplemental.csv

```diff
@@ -13,6 +13,21 @@ Lucira COVID-19 All-In-One Test Kit,96986-5,00810055970001,,N,UNK,N,UNK,https://
 Test_OTC_Device,,Test_OTC_Device,Test_OTC_Device,Y,,,,,,T
 Test_Home_Device,,Test_Home_Device,Test_Home_Device,,Y,,,,,T
 Test_Unproctored_Device,,Test_Unproctored_Device,Test_Unproctored_Device,,,,Y,,,T
-flowflex,97097-0,"Flowflex COVID-19 Antigen Home Test_ACON Laboratories, Inc",,,,,,,,
```

Comment:
Oh I thought they were just repeat values that someone accidentally left in since flowflex and cue both have full entires in the LIVD table. I can add them back in to the supplemental though.